pkg/imagebuilder/content.go

```diff
@@ -33,10 +35,30 @@ func (c ConfigBuilder) Ignition() ([]byte, error) {
 		},
 	}
 
-	config.Storage.Files, err = c.getFiles()
+	files, err := c.getFiles()
 	if err != nil {
 		return nil, err
 	}
+	// pull secret not included in data/ignition/files because embed.FS
+	// does not list directories with name starting with '.'
```

Comment:
For CI we won't be able to check in the pull secret anyway, so this is probably what we want.